web_src/less/_base.less

```diff
@@ -332,6 +332,11 @@ a.commit-statuses-trigger {
   &:extend(.unselectable);
 }
 
+.issue-title code {
+  padding: 2px 4px;
+  border-radius: 6px;
```

Comment:
Why a different border-radius than in `web_src/less/_dashboard.less:145` (`3px`)?

Reply:
The font is larger here.

Reply:
6px is better than 3px